modules/persistence/src/services/metadata/associated_products/index.ts

```diff
@@ -74,10 +82,25 @@ export const hasAssociations = (metadata) => !!metadata.associated_products?.len
 const umbrellaMetadataEntry = async (project: string): Promise<Metadata> => {
   try {
     const snooty = await db();
+
+    // first find any umbrella
+    const umbrella = await snooty.collection('metadata').findOne({
+      'associated_products.name': project,
+      is_merged_toc: false,
+    });
```

Comment:
nit for the future: this should probably return most recent result, but not worth worrying about yet.

Reply:
that's a good catch. adding sort in the findOptions